doc/SUMMARY.md

```diff
@@ -9,9 +9,11 @@
 * [Services and Contributions](Services_and_Contributions.md)
 
 ## Using Theia
+
 * [Build your own IDE](Composing_Applications.md)
 * [Authoring an Extension](Authoring_Extensions.md)
 * [Authoring Plug-ins](Authoring_Plugins.md)
+* [TextMate Support](TextMate.md)
```

Comment:
I find the title a bit ambiguous.  TextMate is a text editor, we don't support running TextMate in Theia :).  Also, it's not clear what this even means for somebody that doesn't know what TextMate grammars are and that you can use them for syntax highlighting.  So I would prefer something like "Syntax highlighting using TextMate grammars".

Reply:
`TextMate Coloring`?